code/modules/mob/inventory.dm

```diff
@@ -77,25 +71,12 @@
 	W.plane = initial(W.plane)
 	W.dropped()
 
-/mob/proc/drop_item_v()		//this is dumb.
-	if(stat == CONSCIOUS && isturf(loc))
-		return drop_item()
-	return 0
-
-//Drops the item in our left hand
-/mob/proc/drop_l_hand()
-	return unEquip(l_hand) //All needed checks are in unEquip

```

Comment:
Why would we have a proc that literally does nothing but call another proc while ignoring its force argument. It's not helping readability while simultaneously hurting functionality.